jvm-packages/xgboost4j-example/src/main/scala/ml/dmlc/xgboost4j/scala/example/XGBoostScalaExample.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ Copyright (c) 2014 by Contributors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+package ml.dmlc.xgboost4j.scala.example
```

Comment:
This file is the example I extracted from https://github.com/dmlc/xgboost/tree/master/jvm-packages. I put here as an runnable example for readers.